libs/components/layout/src/lib/modules/action-button/fixtures/action-button.component.fixture.html

```diff
@@ -3,33 +3,53 @@
     [style.height]="firstButtonHeight"
     (actionClick)="buttonClicked()"
   >
+    @if(useIconName) {
```

Comment:
usually we have a space after the `@if`, not sure if that is something we want to be consistent about? Surprised we don't have prettier formatting around that

Reply:
Fixed. Not sure why prettier didn't do that.